applications/visualizer/frontend/src/models/workspace.ts

```diff
@@ -68,18 +77,33 @@ export class Workspace {
     contexts?: Record<ViewerType, SynchronizerContext>,
     visibilities?: Record<string, ViewerData>,
     neuronGroups?: Record<string, NeuronGroup>,
+    emViewerSettings?: EMViewerSettings,
+    viewers?: Record<ViewerType, boolean>,
   ) {
     this.id = id;
     this.name = name;
     this.activeDatasets = activeDatasets;
     this.availableNeurons = {};
     this.activeNeurons = activeNeurons || new Set();
-    this.viewers = {
+    this.viewers = viewers || {
       [ViewerType.Graph]: true,
       [ViewerType.ThreeD]: false,
       [ViewerType.EM]: false,
     };
     this.neuronGroups = neuronGroups || {};
+    // Set EM viewer settings
+    if (!emViewerSettings) {
+      const firstActiveDataset = Object.values(activeDatasets)?.[0];
+      const [minSlice, maxSlice] = firstActiveDataset.emData.sliceRange;
```

Comment:
When running the application for the first time, I'm getting:
`Uncaught (in promise) TypeError: Cannot read properties of null (reading 'sliceRange')`

![image](https://github.com/user-attachments/assets/3d1d0602-e757-4803-9ebe-b1f76ef9c935)

Reply:
@afonsobspinto Ah, yes, you need to re-run the "ingestion" for the datasets you have, then population (you can run the ingestion in `--dry-run` mode), as there is new metadata information that needs to be generated and inserted in the database. Sorry I forgot to mention that. You'll need to rerun the "ingestion" for the EM files for the datasets you have in local (this would need to be done also for the dev platform/bucket)